softdev.bib

```diff
@@ -2899,3 +2899,70 @@ @inproceedings{shahriyar14fast
     month = Oct,
     pages = "121-139"
 }
+
+@article{martin08common,
+  author = {Martin, Robert A. and Barnum, Sean},
+  title = {Common Weakness Enumeration (CWE) Status Update},
+  year = {2008},
+  issue_date = {April 2008},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  volume = {XXVIII},
+  number = {1},
+  issn = {1094-3641},
+  url = {https://doi.org/10.1145/1387830.1387835},
+  doi = {10.1145/1387830.1387835},
+  journal = {Ada Lett.},
+  month = Apr,
+  pages = {88–91},
+  numpages = {4}
+}
+
+@inproceedings{evans06scalable,
+  title={A scalable concurrent malloc (3) implementation for FreeBSD},
+  author={Evans, Jason},
+  booktitle={BSDCan},
+  year={2006},
+  month=May,
+  url={https://people.freebsd.org/~jasone/jemalloc/bsdcan2006/jemalloc.pdf}
+}
+
+@inproceedings{lietar19snmalloc,
+  author = {Li\'{e}tar, Paul and Butler, Theodore and Clebsch, Sylvan and Drossopoulou, Sophia and Franco, Juliana and Parkinson, Matthew J. and Shamis, Alex and Wintersteiger, Christoph M. and Chisnall, David},
+  title = {Snmalloc: A Message Passing Allocator},
+  year = {2019},
+  month = Jun,
+  isbn = {9781450367226},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  url = {https://doi.org/10.1145/3315573.3329980},
+  doi = {10.1145/3315573.3329980},
+  booktitle = {ISMM},
+  pages = {122–135},
+  numpages = {14},
+  keywords = {Memory allocation, message passing},
+  location = {Phoenix, AZ, USA},
+  series = {ISMM 2019}
+}
+
+@misc{lea96memory,
+  title={A memory allocator},
+  author={Lea, Doug and Gloger, Wolfram},
+  year={1996},
+  url={http://web.mit.edu/sage/export/singular-3-0-4-3.old/omalloc/Misc/dlmalloc/malloc.ps}
```

Comment:
Where was this published?

Reply:
I couldn't find it being published anywhere (except the intro says a German translation appearing in something called "unix/mail" in 1996). However, I did find a proper webpage with the same content as this `.ps` document, so I've replaced it in https://github.com/softdevteam/softdevbib/pull/86/commits/5ebdc5fa93f1a6028041a847d251b1f388c74482.